### ISSUE TITLE
Uncaught TypeError: undefined is not a function

### ISSUE THREAD

**Issue by gmeral** [smashwilson/merge-conflicts#119]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.165.0
**System**: linux 3.13.0-44-generic
**Thrown From**: [merge-conflicts](https://github.com/smashwilson/merge-conflicts) package, v1.2.9

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /home/guillaume/.atom/packages/merge-conflicts/lib/resolver-view.coffee:35

TypeError: undefined is not a function
  at ResolverView.module.exports.ResolverView.relativePath (/home/guillaume/.atom/packages/merge-conflicts/lib/resolver-view.coffee:35:73)
  at ResolverView.module.exports.ResolverView.refresh (/home/guillaume/.atom/packages/merge-conflicts/lib/resolver-view.coffee:38:25)
  at ResolverView.module.exports.ResolverView.initialize (/home/guillaume/.atom/packages/merge-conflicts/lib/resolver-view.coffee:26:6)
  at ResolverView.View (/home/guillaume/.atom/packages/merge-conflicts/node_modules/space-pen/lib/space-pen.js:184:25)
  at new ResolverView (/home/guillaume/.atom/packages/merge-conflicts/lib/resolver-view.coffee:9:3)
  at ConflictMarker.module.exports.ConflictMarker.conflictsResolved (/home/guillaume/.atom/packages/merge-conflicts/lib/conflict-marker.coffee:70:42)
  at /home/guillaume/.atom/packages/merge-conflicts/lib/conflict-marker.coffee:61:10
  at Emitter.module.exports.Emitter.emit (/usr/share/atom/resources/app/node_modules/event-kit/lib/emitter.js:82:11)
  at Object.pkgEmitter.didResolveConflict (/home/guillaume/.atom/packages/merge-conflicts/lib/merge-conflicts.coffee:14:47)
  at /home/guillaume/.atom/packages/merge-conflicts/lib/conflict-marker.coffee:27:14
  at Emitter.module.exports.Emitter.emit (/usr/share/atom/resources/app/node_modules/event-kit/lib/emitter.js:82:11)
  at Conflict.module.exports.Conflict.resolveAs (/home/guillaume/.atom/packages/merge-conflicts/lib/conflict.coffee:115:14)
  at TheirSide.Side.resolve (/home/guillaume/.atom/packages/merge-conflicts/lib/side.coffee:7:25)
  at SideView.module.exports.SideView.useMe (/home/guillaume/.atom/packages/merge-conflicts/lib/side-view.coffee:53:11)
  at HTMLButtonElement.<anonymous> (/home/guillaume/.atom/packages/merge-conflicts/node_modules/space-pen/lib/space-pen.js:220:36)
  at HTMLButtonElement.jQuery.event.dispatch (/home/guillaume/.atom/packages/merge-conflicts/node_modules/space-pen/node_modules/jquery/dist/jquery.js:4430:9)
  at HTMLButtonElement.elemData.handle (/home/guillaume/.atom/packages/merge-conflicts/node_modules/space-pen/node_modules/jquery/dist/jquery.js:4116:28)

```

### Commands

```
     -4:11.7 settings-view:open (input.hidden-input)
     -4:01.2 core:confirm (input.hidden-input)
     -3:23.3 core:backspace (input.hidden-input)
     -1:48.8 merge-conflicts:detect (input.hidden-input)
     -0:13.0 core:copy (input.hidden-input)
     -0:08.6 core:paste (input.hidden-input)
     -0:02.9 core:backspace (input.hidden-input)
 25x -0:01.9 core:move-left (input.hidden-input)
  2x -0:00.0 core:move-right (input.hidden-input)
```

### Config

```
{
  "core": {
    "themes": [
      "seti-ui",
      "seti-syntax"
    ],
    "disabledPackages": [
      "minimap"
    ]
  }
}
```

### Installed Packages

```coffee
# User
linter, v0.9.0
linter-pylint, v0.2.0
merge-conflicts, v1.2.9
seti-syntax, v0.2.1
seti-ui, v0.5.1
wheel-zoom, v0.6.0

# Dev
No dev packages
```

/cc @atom/core

**Comment by krmaxwell** [comment]
Occurred for me when accepting changes during merge conflict resolution.

**Comment by smashwilson** [comment]
Part of the work in #165 might have corrected this. Can you replicate on 1.3.3?

**Comment by Eschon** [comment]
I Just got this in 1.3.3
For me it happened when I clicked the Quit-Button on the merge-conflict panel at the bottom
![screen shot 2015-07-17 at 10 09 50](https://cloud.githubusercontent.com/assets/11647830/8743069/048747bc-2c6c-11e5-99d6-95afe0130998.png)
Every time I click it this happens and it won't disappear.

**Atom Version**: 1.0.2
**System**: Mac OS X 10.10.4
**Thrown From**: [merge-conflicts](https://github.com/smashwilson/merge-conflicts) package, v1.3.3


### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /Users/egon/.atom/packages/merge-conflicts/lib/view/merge-conflicts-view.coffee:122

TypeError: undefined is not a function
  at MergeConflictsView.finish (/Users/egon/.atom/packages/merge-conflicts/lib/view/merge-conflicts-view.coffee:122:7)
  at MergeConflictsView.quit (/Users/egon/.atom/packages/merge-conflicts/lib/view/merge-conflicts-view.coffee:86:6)
  at HTMLButtonElement.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:220:36)
  at HTMLButtonElement.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
  at HTMLButtonElement.elemData.handle (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)

```

### Commands

```
     -4:55.5.0 command-palette:toggle (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -4:53.7.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -4:53.7.0 merge-conflicts:detect (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -4:31.3.0 core:save (atom-text-editor.editor.conflicted.is-focused)
 12x -4:16.8.0 core:backspace (atom-text-editor.editor.is-focused)
     -4:02.4.0 snippets:previous-tab-stop (atom-text-editor.editor.is-focused)
     -4:02.4.0 editor:outdent-selected-rows (atom-text-editor.editor.is-focused)
     -4:00.6.0 snippets:previous-tab-stop (atom-text-editor.editor.is-focused)
     -4:00.6.0 editor:outdent-selected-rows (atom-text-editor.editor.is-focused)
     -3:59.3.0 core:save (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "one-dark-ui",
      "solarized-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-handlebars, v1.0.0
atom-ternjs, v0.6.8
linter, v1.2.4
linter-clojure, v0.0.5
linter-csslint, v0.0.14
linter-jscs, v1.12.0
linter-jshint, v1.0.5
linter-rubocop, v0.2.7
linter-xmllint, v0.0.7
merge-conflicts, v1.3.3
pretty-json, v0.4.1

# Dev
No dev packages
```

**Comment by georgiana-gligor** [comment]
Confirmed with 1.3.3 here as well.
The "Stage" label present as well on the file causing trouble.

**Comment by smashwilson** [comment]
> Comparing the stack traces I realised that this is probably a completely different error should I open another issue?

@Eschon Ah, yes, I think I see what's going on in that one, and it is a different issue. If you see this before I get to a fix, a new issue would be fine :smile:

> Confirmed with 1.3.3 here as well.
> The "Stage" label present as well on the file causing trouble.

@georgiana-gligor Can you double-check which stack trace you're seeing - @Eschon's, or @gmeral's? Basically look for `MergeConflictsView` or `ResolverView` in the top few stack frames.

**Comment by smashwilson** [comment]
Looking at the original stack, #165 (released as part of v1.3.3) should indeed deal with this. If you can still replicate on 1.3.4, let me know :grin:
